index.bs

```diff
@@ -1637,8 +1645,8 @@ each [=sensor type=] in [=extension specifications=]:
     Its [=attributes=] which expose [=sensor readings=] are [=read only=] and
     their getters must return the result of invoking [=get value from latest reading=]
     with <strong>this</strong> and [=attribute=] [=identifier=] as arguments.
--   A {{PermissionName}}, if the [=sensor type=] is not representing
-    [=sensor fusion=] (otherwise, {{PermissionName|PermissionNames}}
+-   A [=permission name=], if the [=sensor type=] is not representing
```

Comment:
Will you submit PRs to concrete sensor specs accordingly?

Reply:
yeah,  I'll do it.